ts/screens/IngressScreen.tsx

```diff
@@ -43,7 +47,24 @@ class IngressScreen extends React.PureComponent<Props> {
   public componentDidMount() {
     // Dispatch START_APPLICATION_INITIALIZATION to initialize the app
     this.props.dispatch(startApplicationInitialization());
```

Comment:
@cloudify @Edigio are we sure we want to start the applicationInizialization also if the device is rooted? I think it must be done only if not rooted.

Reply:
agreed

Reply:
Ok, this is the result:
![IMG_0225](https://user-images.githubusercontent.com/7993551/56223033-4bbead80-606d-11e9-9bb1-b27e18c467b1.PNG)